static/js/subs.js

```diff
@@ -575,6 +575,10 @@ exports.setup_page = function (callback) {
             },
         });
 
+        if (page_params.is_guest) {
+            exports.toggler.disable_tab('all-streams');
+        }
+
```

Comment:
We also need to remove unsubscribed streams from rendering into `All streams` tabs along with this change. Cause if guest user tries access All stream tab(by removing `disabled` class) guest user is able to see all the list of streams.